README.md

```diff
@@ -1,6 +1,106 @@
+# Logstash Filter Aggregate Documentation
```

Comment:
This is very nice and should be added to more other plugins, the others plugins are just using a default README :+1: 

Do you think mgith be this section with examples is better in other place of the document? but In general I like